src/helpers/errorHandler.js

```diff
@@ -1,14 +1,21 @@
 import R from 'ramda';
 import { openResponseModal } from '../store/actions';
 
-const errorHandler = dispatch => error => R.compose(
-    dispatch,
-    openResponseModal,
-)({
-    // Show error message from server if there is one
-    message: error.response.data.error ?
-             error.response.data.error :
-             `${error}: ${error.response.data}`,
-});
+const generateMessage = error => {
```

Comment:
What about this style?
```js
const generateMessage = error => {
  if (!error.response) return `${error}`;

  switch true {
    case error.response.data && error.response.data.error:
      return `${error} ${error.response.data.error}`;
    case error.response.data:
      return `${error} ${error.response.data}`;
    case error.response:
      return `${error} ${error.response}`;
  }
};

const errorHandler = (dispatch, error) =>
  R.compose(
    dispatch,
    openResponseModal,
    generateMessage
  )(error)
```

Reply:
@s4nji @rafaeljesus 
I think we do not need this at all. Before it was needed, becase sometimes there was case when `error` didn't contain `.response` property. But after latest [fix](https://github.com/hellofresh/janus-dashboard/pull/228) we do not receiving `error` without _response_. It was happening because of bad logic in the interceptor.

Reply:
Cool @ivanzusko 👍